### PR TITLE
[FEAT] data parse store and monitor syslog data

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: configurations/aurora/ProviderMonitoringNetworkForeignBus.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	"schema.registry.url": "http://admin:8081",
 	"value.deserializer": "io.confluent.kafka.serializers.KafkaAvroDeserializer",
 	"auto.commit.enable": true,
-        "topics": "sensors_node"
+        "topics": "sensors_node,SYSLOG"
```
Comment (paul-amonson) [inline configurations/aurora/ProviderMonitoringNetworkForeignBus.json:15]: This is a Ras Event not a sensor telemetry. Use the section with "sel_log".

File: monitoring/src/main/java/com/intel/dai/monitoring/EnvironmentalProviderHPCM.java [hunk 2]
```diff
@@ -141,6 +141,7 @@ private void setUpConfig(NetworkListenerConfig config) {
             processorMap_.put("sensors_node", new TopicMetricSensorNode(log_, doAggregation));
             processorMap_.put("metric_cooldev", new TopicMetricCoolDev(log_, doAggregation));
             processorMap_.put("pcm-monitoring", new TopicMetricPcmMonitoring(log_, doAggregation));
+            processorMap_.put("SYSLOG", new TopicMetricSyslog(log_, doAggregation));
```
Comment (paul-amonson) [inline monitoring/src/main/java/com/intel/dai/monitoring/EnvironmentalProviderHPCM.java:144]: Same here: This is a Ras Event not a sensor telemetry. Use the section with "sel_log".

File: monitoring/src/main/java/com/intel/dai/monitoring/TopicMetricSyslog.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
+package com.intel.dai.monitoring;
```
Comment (paul-amonson) [inline monitoring/src/main/java/com/intel/dai/monitoring/TopicMetricSyslog.java:1]: Model after the parser for RasEvents not Metrics.

**Review by paul-amonson — CHANGES_REQUESTED** [review]
Address overall design issue.

File: configurations/aurora/ProviderMonitoringNetworkForeignBus.json (hunk 1 above)
Comment (sindhu-106) [reply configurations/aurora/ProviderMonitoringNetworkForeignBus.json]: moved to ras event monitoring

**Comment by sindhu-106** [comment]
> Address overall design issue.

made changes you suggested.

File: monitoring/src/main/java/com/intel/dai/monitoring/TopicEventSyslog.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                final long timestamp = TimeUtils.nSFromIso8601(map.getString(TIMESTAMP));
+                final PropertyMap hostInfoMap = map.getMapOrDefault(HOST, new PropertyMap());
+                final String location = hostInfoMap.getStringOrDefault(NAME, "");
+                final PropertyMap syslogDataMap = map.getMapOrDefault(data.topic.toLowerCase(), new PropertyMap());
+                final long priority = syslogDataMap.getLongOrDefault(PRIORITY, 3);
+                final boolean triggerEvent = triggerEvent(priority);
+                if(triggerEvent)
+                    addToResults(data.topic, RAS_EVENT_NAME, data.originalJsonText, location, timestamp, results);
+            } catch (Exception e) {
+                log_.warn("Error: ", e.getMessage());
```
Comment (paul-amonson) [inline monitoring/src/main/java/com/intel/dai/monitoring/TopicEventSyslog.java:34]: log_warn("Error: %s", e.getMessage());

**Review by paul-amonson — DISMISSED** [review]

File: monitoring/src/main/java/com/intel/dai/monitoring/TopicEventSyslog.java (hunk 4 above)
Comment (sindhu-106) [reply monitoring/src/main/java/com/intel/dai/monitoring/TopicEventSyslog.java:34]: good catch Paul. Will update it :)

**Review by avincigu — APPROVED** [review]

**Review by paul-amonson — APPROVED** [review]

**Pull request merged**